### DISCUSSION TITLE
How to delete all cookies while running pywebview app using APIs

### DISCUSSION THREAD

**Discussion by matphilips90** [r0x0r/pywebview#1359] [Q&A]
We are stuck as we are not able to delete the cookies of the google sso page. Due to this our app keeps the session of the user logged in last time. Any help would be highly appreciated.

Is there a way to call clear all the cookies in python

**Comment by r0x0r** [comment]
Incognito mode is on by default for _pywebview_. It should not remember logged in sessions after app restart. Do you get a different behaviour?

**Reply by matphilips90** [reply]
Hi @r0x0r  actually our application runs entire day so it maintain the login sessions until you stop the application.

**Reply by r0x0r** [reply]
Can you delete cookies with Javascript? `window.evaluate_js('document.cookie=""')` or something along these lines?

**Reply by matphilips90** [reply]
Yes we tried it but  can't delete httpOnly cookies with JavaScript

**Reply by r0x0r** [reply]
I see.
Deleting cookies could be implemented in Python. I am not sure what API should look like, nor when I have time to tackle it.
If you want to sponsor this issue or implement it yourself, let me know.

**Reply by matphilips90** [reply]
HI @r0x0r  can you share your email address we can have a chat on this

**Reply by r0x0r** [reply]
roman@flowrl.com
